app/views/interactive_pages/_show.html.haml

```diff
@@ -11,8 +11,8 @@
     - if page.visible_interactives.length > 0 && page.show_interactive
       = render partial: 'interactive_pages/interactive', locals: {page: page, layout: layout}
 
-  .questions-mod.ui-block-2{ :class => page.embeddable_display_mode == 'carousel' ? 'jcarousel' : '' }
-    - if page.show_info_assessment
+  - if page.show_info_assessment
+    .questions-mod.ui-block-2{ :class => page.embeddable_display_mode == 'carousel' ? 'jcarousel' : '' }
```

Comment:
What if the layout is 60-40, but the user choses to not show the info_assessment block.  Currently LARA will still keep the interactive at 60% width. I don't think this matters if the result is a 60% interactive or a 100% interactive, but I just wanted to make sure this change doesn't break the page in that case.

Reply:
Yes, I was testing it. And double checked now. Interactive will still take 60 or 70% space only. I think it makes sense, if the author wants full width then the full-width layout is there (or responsive).